src/OrchardCore/OrchardCore.ContentManagement/DefaultContentManager.cs

```diff
@@ -596,19 +598,19 @@ protected async Task<IEnumerable<ContentItem>> BuildNewVersionsAsync(IEnumerable
 
         public async Task CreateAsync(ContentItem contentItem, VersionOptions options)
         {
-            if (string.IsNullOrEmpty(contentItem.ContentItemVersionId))
-            {
-                contentItem.ContentItemVersionId = _idGenerator.GenerateUniqueId(contentItem);
-                contentItem.Published = true;
-                contentItem.Latest = true;
-            }
+            contentItem.Published = true;
 
             // Draft flag on create is required for explicitly-published content items
             if (options.IsDraft)
             {
                 contentItem.Published = false;
             }
 
+            if (string.IsNullOrEmpty(contentItem.ContentItemVersionId))
+            {
+                contentItem.ContentItemVersionId = _idGenerator.GenerateUniqueId(contentItem);
+            }
+
```

Comment:
We still set the ContentItemVersionId on really edge use cases that don't rely on the defaultManager entirely.